scripts/run_local.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+mkdir -p models
```

Comment:
Should this be created after the `cd` on line 4 so that we don't have random model dirs floating around if we run it in different dirs?

Reply:
Good catch!